### DISCUSSION TITLE
Auto Arima is not working

### DISCUSSION THREAD

**Discussion by Abhaysharma53** [Nixtla/statsforecast#217] [Q&A] [answered]
So, I am really new to this forecasting library, and i am facing some issue to run the auto_arima function of it.
so Basically, i tested the statsforecast model on python 3.9 and it was working fine, but due to a project requirement right now i am using it in the virtual environment with python 3.8 , and i am facing this issue "
ImportError: cannot import name 'auto_arima' from 'statsforecast.models'
", can anyone help me to understand what i might be doing wrong?

**Comment by AzulGarza** [comment, marked as answer]
Hi @Abhaysharma53!

The problem probably arises from the update of the models that we made from version 1.0.0. Before that version, the models followed a functional paradigm. The most recent version uses classes with `fit` and `predict` methods (like sklearn). So, to import auto arima the new line would be:

```
from statsforecast.models import AutoARIMA
```

You can find an example of the new syntaxis here: https://nixtla.github.io/statsforecast/examples/Getting_Started_with_Auto_Arima_and_ETS.html.
